xrootd/resources/xrootd-cache.cfg

```diff
@@ -62,10 +62,14 @@ throttle.throttle concurrency {{.Cache.Concurrency}}
 pss.origin {{.Cache.PSSOrigin}}
 # FIXME: the oss.space meta / data only works if the meta and data directories are different physical devices.
 # Otherwise, no data space is setup and the cache simply doesn't write out data.
-oss.localroot {{.Cache.DataLocation}}
-#pfc.spaces data meta
-#oss.space meta {{.Cache.DataLocation}}/meta*
-#oss.space data {{.Cache.DataLocation}}/data*
+oss.localroot {{.Cache.LocalRoot}}
+pfc.spaces data meta
```

Comment:
Based on the comment above explaining that this only works if `meta/data` are different devices, I'm worried about someone setting the deprecated `Cache.DataLocation` that now enables this previously-disabled feature. Perhaps we can keep the behavior of `Cache.DataLocation` the same as it was and not have any defaults for these values?

Reply:
It seems like that above comment isn't correct anymore. I was able to get this to work on the same device. I'm unsure why I put that there originally.

Reply:
Talked with BrianB about this. We've decided it's probably an erroneous comment.